DigitalLearningSolutions.Data.Tests/DataServices/CentresDataServiceTests.cs

```diff
@@ -125,5 +126,33 @@ public void GetCentreName_should_return_null_when_the_centre_does_not_exist()
             // Then
             result.Should().BeNull();
         }
+
+        [Test]
+        public void UpdateCentreManagerDetails_updates_centre()
+        {
+            using var transaction = new TransactionScope();
+            try
+            {
+                // Given
+                var firstName = "TestFirstName";
+                var lastName = "TestLastName";
+                var email = "test@email.com";
+                var telephone = "0123456789";
+
+                // When
+                centresDataService.UpdateCentreManagerDetails(2, firstName, lastName, email, telephone);
+                var updatedCentre = centresDataService.GetCentreDetailsById(2);
+
+                // Then
+                updatedCentre.ContactForename.Should().BeEquivalentTo(firstName);

```

Comment:
Might be worth wrapping these in an AssertionScope so it checks them all on failure